# STAT110 kapittel 9.1-2
Saltholdighet blir målt på en rekke målepunkter i Norskehavet og Barentshavet. Kystvann har en lavere saltholdighet enn atlantisk vann, som blir fraktet inn av Golfstrømmen. Ved en saltholdighet på over 35 promille regnes vannet som atlantisk.
    I et visst geografisk område et stykke fra kysten ble følgende målinger av saltholdighet i overflaten gjort 
   $$x_i:35.3, 35.8, 35.6, 36.3, 36.1, 34.9, 37.1, 36.2, 34.7, 35.5, 35.8, 35.3$$

b) Før målingene ble foretatt mente man at forventet saltholdighet i dette området er større enn 35. Test nullhypotesen $\mu = 35$ mot $H_1:\mu > 35$ ved et signifikansnivå på 0.05. 

### Løsning: 
Bruker formelen:
$t=\frac{\bar{x}-\mu_0}{s_x/\sqrt{n}}$ for t-observator.


In [50]:
import numpy as np
from scipy import stats

x=[35.3, 35.8, 35.6, 36.3, 36.1, 34.9, 37.1, 36.2, 34.7, 35.5, 35.8, 35.3]
n=len(x)
x_mean= np.mean(x)
mu_0= 35
Sx= stats.tstd(x)   #standardavvik

t_11_05=np.round(stats.t.ppf(0.95, 11),3)  #i stedet for tabell A5
print("Kritisk verdi:", t_tab)

#t-observator
t= (x_mean-mu_0)/(Sx/(np.sqrt(n)))    
print("Observert t:",tset)

#python sin innebygde t-observator
tset,pval= stats.ttest_1samp(x,35)
print("Observert t (innebygget) :", tset)

#Utfører ensidig hypotesetest
if t> t_11_05:
    print("Forkast H0")
else:
    print("Behold H0")


Kritisk verdi: 1.796
Observert t: 3.77531167933242
Observert t (innebygget) : 3.77531167933242
Forkast H0


Ser at vi får samme svar for t-observatoren ved å løse oppgaven rett frem og ved å bruke python(Scripy) sin innebygde metode. Vi forkaster null hypotesen fordi 3.78> 1.796.


